web/src/containers/BudgetSummary.test.js

```diff
@@ -40,7 +40,7 @@ describe('budget summary component', () => {
         <DataRow
           category="category text"
           data={{
-            total: { total: 0 }
+            total: { medicaid: 0, federal: 0, state: 0, total: 0 }
```

Comment:
Since we're not testing invalid data, make sure the data is valid.  😛  This gets rid of some warnings about the `<Dollars>` component